api/src/uvisor-input.S

```diff
@@ -93,14 +91,23 @@ uvisor_config:
     /* Privileged system IRQ hooks */
     .long __uvisor_priv_sys_irq_hooks
 
+/* uVisor mode of operation
+ * 0: Disabled
+ * 1: Permissive
+ * 2: Enabled */
```

Comment:
What if we change these values elsewhere? We'd forget to update this comment, probably. For the sake of maintainability, consider removing the values here and reference `enum` names or `#define` names instead, if you want to reference the values that `__uvisor_mode` can be.